features/step_definitions/inbox_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given(/^I am on the welcome page$/) do
+  visit welcome_index_path
+end
+
+Then(/^I should be on inbox page$/) do
+  visit mailbox_inbox_path
+end
+
+Then(/^I send a mail from "([^"]*)" to "([^"]*)"$/) do |name1, name2|
+  @user = User.find_by(name: name1)
+  @receiver = User.find_by(name: name2)
+  @user.send_message(@receiver, 'Lorem ipsum...', 'Subject')
+end
```

Comment:
Good. You can use 'Lorem ipsum...' to find out if the message is actually on the page when user2 views it.